From b469fa3b688c785774f828f3d346bc39d8c07506 Mon Sep 17 00:00:00 2001
From: "Robert T. McGibbon" <rmcgibbo@gmail.com>
Date: Mon, 18 May 2020 13:48:48 -0400
Subject: [PATCH 1/1] Add 38 and 39 to sconsutils

---
 external/sconsutils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/external/sconsutils.py b/external/sconsutils.py
index 6d95e53..bcd0792 100644
--- a/external/sconsutils.py
+++ b/external/sconsutils.py
@@ -287,7 +287,7 @@ def generate(env):
     opts.Add("OBJDIR", "build product location", 'build')
     opts.Add("PREFIX", "installation location")
 
-    opts.Add(ListVariable('PYTHONVER', 'python versions', os.getenv('PYTHONVER', ''), ['27', '35', '36', '37']))
+    opts.Add(ListVariable('PYTHONVER', 'python versions', os.getenv('PYTHONVER', ''), ['27', '35', '36', '37', '38', '39]))
     opts.Update(env)
 
     builddir = env.Dir(env['OBJDIR']).srcnode().abspath
-- 
2.23.0

